src/TLuaInterpreter.cpp

```diff
@@ -12231,6 +12231,21 @@ int TLuaInterpreter::ttsSpeak(lua_State* L)
     QString textToSay;
     textToSay = lua_tostring(L, 1);
 
+    textToSay = textToSay.trimmed();
+    if (textToSay.isEmpty()) { // there's nothing more to say. discussion: https://github.com/Mudlet/Mudlet/issues/4688
+        return warnArgumentValue(L, __func__, QStringLiteral("skipped empty text to speak (TTS)"));
+    }
+
+    std::vector<QString> dontSpeak = {"<", ">", "&lt;", "&gt;"}; // discussion: https://github.com/Mudlet/Mudlet/issues/4689
+    for (QString dropThis : dontSpeak) {
```

Comment:
```suggestion
    for (const auto dropThis : dontSpeak) {
```

Reply:
Not sure I like auto over QString here. It muddles the clear water. Const seams fine though.

Reply:
Fair call.